src/plugins/advanced_settings/public/management_app/mount_management_section.tsx

```diff
@@ -72,26 +73,29 @@ export async function mountManagementSection(
 
   ReactDOM.render(
     <KibanaThemeProvider theme$={params.theme$}>
-      <I18nProvider>
-        <Router history={params.history}>
-          <Switch>
-            {/* TODO: remove route param (`query`) in 7.13 */}
-            <Route path={`/:${QUERY}`}>{(props) => <Redirect to={redirectUrl(props)} />}</Route>
-            <Route path="/">
-              <AdvancedSettings
-                history={params.history}
-                enableSaving={canSave}
-                toasts={notifications.toasts}
-                docLinks={docLinks.links}
-                uiSettings={uiSettings}
-                theme={params.theme$}
-                componentRegistry={componentRegistry}
-                trackUiMetric={trackUiMetric}
-              />
-            </Route>
-          </Switch>
-        </Router>
-      </I18nProvider>
+      <KibanaContextProvider services={{ settings, notifications, docLinks, application, chrome }}>
+        <I18nProvider>
+          <Router history={params.history}>
+            <Switch>
+              {/* TODO: remove route param (`query`) in 7.13 */}
+              <Route path={`/:${QUERY}`}>{(props) => <Redirect to={redirectUrl(props)} />}</Route>
+              <Route path="/">
+                <Settings
+                  history={params.history}
+                  enableSaving={canSave}
+                  toasts={notifications.toasts}
+                  docLinks={docLinks.links}
+                  uiSettings={settings.client}
+                  globalUiSettings={settings.globalClient}
```

Comment:
NIT: shouldn't we pass `settings: SettingsStart` instead of passing both clients individually?